<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225,225,225); font-size: 28px;</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupusernameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225,225,225); font-size: 15px;
</string>
   </property>
   <property name="text">
    <string>Username: </string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmpPasswordLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225,225,225); font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupUsernameForm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="SignupConfirmPasswordForm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupSubmitButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey; font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="signupPasswordLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225,225,225); font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SignupPasswordForm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupReturnButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey; font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>355</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225,225,225); font-size: 12px;</string>
   </property>
   <property name="text">
    <string>FEEDBACK INFORMATION HERE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
